cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -229,7 +229,7 @@ func (s *VMTServer) AddFlags(fs *pflag.FlagSet) {
 	// So far we have noticed cluster api support only in openshift clusters and our implementation works only for openshift
 	// It thus makes sense to have openshifts machine api namespace as our default cluster api namespace
 	fs.StringVar(&s.ClusterAPINamespace, "cluster-api-namespace", "openshift-machine-api", "The Cluster API namespace.")
-	fs.StringVar(&s.BusyboxImage, "busybox-image", "busybox", "The complete image uri used for fallback node cpu frequency getter job.")
+	fs.StringVar(&s.BusyboxImage, "busybox-image", "icr.io/cpopen/turbonomic/cpufreqgetter", "The complete image uri used for fallback node cpu frequency getter job.")
```

Comment:
Can you search for other references to busybox in kubeturbo code?

Reply:
I did.  that's all of places which is referring to `busybox`,  some of them are test code, and others are variable names. do you want me to update all of the variable names and get rid of `busybox` totally?  if so, do we also want to change the args `busybox-image` for the kubeturbo or just remove it?
```
cmd/kubeturbo/app/kubeturbo_builder.go:
  160  
  161: 	// Busybox image uri used for cpufreq getter job
  162: 	BusyboxImage string
  163  
  164  	// Name of the secret that stores the image pull credentials of cpu freq getter job image
  165: 	BusyboxImagePullSecret string
  166  

  231  	fs.StringVar(&s.ClusterAPINamespace, "cluster-api-namespace", "openshift-machine-api", "The Cluster API namespace.")
  232: 	fs.StringVar(&s.BusyboxImage, "busybox-image", "icr.io/cpopen/turbonomic/cpufreqgetter", "The complete image uri used for fallback node cpu frequency getter job.")
  233: 	fs.StringVar(&s.BusyboxImagePullSecret, "busybox-image-pull-secret", "", "The name of the secret that stores the image pull credentials for busybox image.")
  234  	fs.StringVar(&s.CpufreqJobExcludeNodeLabels, "cpufreq-job-exclude-node-labels", "", "The comma separated list of key=value node label pairs for the nodes (for example windows nodes) to be excluded from running job based cpufrequency getter.")

  282  func (s *VMTServer) CreateKubeletClientOrDie(kubeConfig *restclient.Config, fallbackClient *kubernetes.Clientset,
  283: 	busyboxImage, imagePullSecret string, cpufreqJobExcludeNodeLabels map[string]set.Set, useProxyEndpoint bool) *kubeclient.KubeletClient {
  284  	kubeletClient, err := kubeclient.NewKubeletConfig(kubeConfig).

  288  		// Timeout(to).
  289: 		Create(fallbackClient, busyboxImage, imagePullSecret, cpufreqJobExcludeNodeLabels, useProxyEndpoint)
  290  	if err != nil {

  415  	}
  416: 	kubeletClient := s.CreateKubeletClientOrDie(kubeConfig, kubeClient, s.BusyboxImage,
  417: 		s.BusyboxImagePullSecret, excludeLabelsMap, s.UseNodeProxyEndpoint)
  418  	caClient, err := clusterclient.NewForConfig(kubeConfig)

pkg/kubeclient/cpufrequency.go:
   42  	kubeClient      *kubernetes.Clientset
   43: 	busyboxImage    string
   44  	imagePullSecret string

   80  // NewNodeCpuFrequencyGetter creates an instance of the abstract type
   81: func NewNodeCpuFrequencyGetter(kubeClient *kubernetes.Clientset, busyboxImage, imagePullSecret string) *NodeCpuFrequencyGetter {
   82  	return &NodeCpuFrequencyGetter{
   83  		kubeClient:      kubeClient,
   84: 		busyboxImage:    busyboxImage,
   85  		imagePullSecret: imagePullSecret,

  294  							Name:            "cpufreq",
  295: 							Image:           n.busyboxImage,
  296  							ImagePullPolicy: "IfNotPresent",

pkg/kubeclient/kubelet_client_test.go:
  71  
  72: 	kc, _ := conf.Create(nil, "busybox", "", map[string]set.Set{}, false)
  73  	entry := &CacheEntry{}

pkg/kubeclient/kubelet_client.go:
  466  
  467: func (kc *KubeletConfig) Create(fallbackClient *kubernetes.Clientset, busyboxImage, imagePullSecret string,
  468  	excludeLabelsMap map[string]set.Set, useProxyEndpoint bool) (*KubeletClient, error) {

  490  		cache:                       make(map[string]*CacheEntry),
  491: 		fallbkCpuFreqGetter:         NewNodeCpuFrequencyGetter(fallbackClient, busyboxImage, imagePullSecret),
  492  		cpufreqJobExcludeNodeLabels: excludeLabelsMap,

test/integration/action_execution.go:
  659  		Name:    name,
  660: 		Image:   "busybox",
  661  		Command: []string{"/bin/sh"},

test/integration/discovery.go:
    93  			s := app.NewVMTServer()
    94: 			kubeletClient := s.CreateKubeletClientOrDie(kubeConfig, kubeClient, "", "busybox", map[string]set.Set{}, true)
    95  

  1004  							Name:    "cont-test-app-mc-1",
  1005: 							Image:   "busybox",
  1006  							Command: []string{"/bin/sh"},

  1010  							Name:    "cont-test-app-mc-2",
  1011: 							Image:   "busybox",
  1012  							Command: []string{"/bin/sh"},

  1079  							Name:    "cont-test-app-mc-wr-1",
  1080: 							Image:   "busybox",
  1081  							Command: []string{"/bin/sh"},

  1095  							Name:    "cont-test-app-mc-wr-2",
  1096: 							Image:   "busybox",
  1097  							Command: []string{"/bin/sh"},

test/integration/garbage_collection.go:
  174  					Name:    "test-cont",
  175: 					Image:   "busybox",
  176  					Command: []string{"/bin/sh"},

test/integration/orm_action_execution.go:
  86  			s := app.NewVMTServer()
  87: 			kubeletClient := s.CreateKubeletClientOrDie(kubeConfig, kubeClient, "", "busybox", map[string]set.Set{}, true)
  88  			apiExtClient, err := apiextclient.NewForConfig(kubeConfig)
```

Reply:
Yes, please remove these references

Reply:
I've removed all of the references and only kept 2 for backward compatibility.